offline.appcache

```diff
@@ -0,0 +1,44 @@
+CACHE MANIFEST
+# 2016-01-14 v1.1.3
+
+# Main HTML file
+index.html
+
+# Translations
+cz/index.html
+da/index.html
+de/index.html
+es/index.html
+fr/index.html
+ko/index.html
+pl/index.html
+pt/index.html
+ru/index.html
+zh/index.html
+
+# CSS Files
+css/critical.css
+css/full.css
+css/fonts/data-ttf.css
+css/fonts/data-woff.css
+css/fonts/data-woff2.css
+
+# JS Files
+assets/js/main.min.js
+assets/js/picturefill.min.js
+
+# Images
+assets/images/css-order-chart.png
```

Comment:
Haha, I can't tell you that!

I know, but I think we might fine. The size limit is 5MB according to http://grinninggecko.com/2011/02/24/developing-cross-platform-html5-offline-app-1/ and the image is 127KB.